### DISCUSSION TITLE
Serial Pins

### DISCUSSION THREAD

**Discussion by Hello-dot-Jpg** [nk64/GT4286#6] [Show and tell]
I've traced the serial pins here off a miyoomini 

![tracks](https://github.com/user-attachments/assets/a814be2f-65f5-48ef-9b4e-bc8f0bc1af17)

Which matches the pin out provided by https://linux-chenxing.org/infinity2/pinouts.html

![image](https://github.com/user-attachments/assets/595aa5b2-5629-4543-b89b-735d2940145a)

On the Retro arcade board things get a little confusing

![tracks2](https://github.com/user-attachments/assets/25bce3f7-b5f6-4977-aff6-0d1f23abd6f0)

I might go out today and grab another device to see if I can add serial to it

**Comment by nk64** [comment]
I bought the last two in my local store but they were for friends so I had to order a third one for myself.

**Reply by Hello-dot-Jpg** [reply]
Wow it looks like I was lucky to get another one, everywhere but my local store now seems out of stock.

**Reply by Hello-dot-Jpg** [reply]
Ok so after a bit of social engineering I managed to get the pin out for the SOC not the SSD202D but the actual SSD203D. I won't share it here but will say that the SSD203D is even less featured than the SSD202D it does not have ethernet as the SSD202D does. :/

I legitimately have no Idea why but engineers who made the PCB are treating it like fort Knox for some reason. Like actual security devices don't go to the extremes these guys have on the hardware side.

All of the NC pins are UARTS or the third USB bus (which is default configured for use with a USB wifi device)

I have also tried using the USB ports to get the device to detect in windows (According to sigmastar documentation it should detect as an adb device) to see if we could get a shell using ADB shell however I haven't been able to get it to be detected by my PC.

I haven't given up yet though as a work in progress I am trying to drill through the PCB up to the bottom of the SOC to expose the pads on the soc for the PM_UART. I have no idea how successful this will be however.

For future reference:
PM_UART_RX 37
PM_UART_TX 38
UART1 RX 41
UART1 TX 42
FUART RX 55
FUARTTX 56
CTS 57
RTS 58
DM_P3 109
DP_P3 110

(use SSD202D doc for chip pin layout)
![IMG_20250103_164136](https://github.com/user-attachments/assets/b029c7e1-bae6-45c8-aa5a-dec8f86bb054)

![IMG_20250103_164256](https://github.com/user-attachments/assets/45b75cb0-4962-45c6-9fc0-0863f50dd136)

**Reply by nk64** [reply]
> Ok so after a bit of social engineering I managed to get the pin out for the SOC not the SSD202D but the actual SSD203D

Nice work

> I legitimately have no Idea why but engineers who made the PCB are treating it like fort Knox for some reason. Like actual security devices don't go to the extremes these guys have on the hardware side.

😆

**Comment by Hello-dot-Jpg** [comment]
Mistakes were made.

![WIN_20250108_21_35_20_Pro](https://github.com/user-attachments/assets/f9a043e9-ef9b-4e0b-8b9e-34eb1888fef3)

I'll use this as a practice one now lol

it was still working before I removed the IC too lol

**Reply by horsiezoo** [reply]
![IMG_2077](https://github.com/user-attachments/assets/7cbdd773-223e-4717-9d68-33b21ee380dd)
Durn ! Did you use a hot air gun thing? Oh well you could get a pandora board for ~$50 and put it in the console 

oh yes almost forgot, the hi-lighted solder pad in this photo, any idea what that connects to? Gnd ?
